### PR TITLE
EM-5085 add vh connection configs

### PR DESCRIPTION
**Pull request by mdayican**
EM-5085 add vh connection configs

### CODE REVIEW

**Review by Jefex7 — APPROVED** [review]

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -114,6 +114,9 @@ azure:
     cvp:
       connection-string: ${CVP_STORAGE_CONNECTION_STRING:DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1}
       blob-container-reference: ${CVP_STORAGE_CONTAINER_NAME:cvptestcontainer}
+    vh:
```
Comment (yogesh-hullatti) [inline src/main/resources/application.yaml:117]: copy paste error? Should they vh_STORAGE_CONTAINER_NAME

File: src/functionalTest/resources/application.yml [hunk 2]
```diff
@@ -69,6 +69,10 @@ azure:
       container-url: ${CVP_STORAGE_URL:https://emhrsapiaat.blob.core.windows.net/cvpsimulator/}
       connection-string: ${CVP_STORAGE_CONNECTION_STRING:DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1}
       blob-container-reference: ${CVP_STORAGE_CONTAINER_NAME:cvptestcontainer}
+    vh:
+      container-url: ${CVP_STORAGE_URL:https://emhrsapiaat.blob.core.windows.net/cvpsimulator/}
+      connection-string: ${CVP_STORAGE_CONNECTION_STRING:DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1}
+      blob-container-name: ${CVP_STORAGE_CONTAINER_NAME:cvptestcontainer}
```
Comment (yogesh-hullatti) [inline src/functionalTest/resources/application.yml:75]: Should this be vhtestcontainer?

File: charts/em-hrs-api/values.yaml [hunk 3]
```diff
@@ -20,6 +20,10 @@ java:
           alias: CVP_STORAGE_CONNECTION_STRING
         - name: cvp-storage-container-name
           alias: CVP_STORAGE_CONTAINER_NAME
+        - name: cvp-storage-connection-string
+          alias: vh_STORAGE_CONNECTION_STRING
```
Comment (yogesh-hullatti) [inline charts/em-hrs-api/values.yaml:24]: Should this be VH_ ? and not vh_

**Review by yogesh-hullatti — CHANGES_REQUESTED** [review]

File: charts/em-hrs-api/values.yaml [hunk 4]
```diff
@@ -20,6 +20,10 @@ java:
           alias: CVP_STORAGE_CONNECTION_STRING
         - name: cvp-storage-container-name
           alias: CVP_STORAGE_CONTAINER_NAME
+        - name: cvp-storage-connection-string
```
[the remark this answers is not in the document]
Comment (mdayican) [reply charts/em-hrs-api/values.yaml:23]: for now using same URL. I will change the urls later

File: src/functionalTest/resources/application.yml [hunk 5]
```diff
@@ -69,6 +69,10 @@ azure:
       container-url: ${CVP_STORAGE_URL:https://emhrsapiaat.blob.core.windows.net/cvpsimulator/}
       connection-string: ${CVP_STORAGE_CONNECTION_STRING:DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1}
       blob-container-reference: ${CVP_STORAGE_CONTAINER_NAME:cvptestcontainer}
+    vh:
```
[the remark this answers is not in the document]
Comment (mdayican) [reply src/functionalTest/resources/application.yml:72]: vh test containers not created yet.

File: src/main/resources/application.yaml (hunk 1 above)
Comment (mdayican) [reply src/main/resources/application.yaml:117]: for now suing same url and containers

Comment (mdayican) [reply src/main/resources/application.yaml:117]: fixed

File: src/functionalTest/resources/application.yml (hunk 2 above)
Comment (mdayican) [reply src/functionalTest/resources/application.yml:75]: for now using cvp , VH vars not created in jenkins and  tests env.

**Pull request merged**